core/kernel.rbs

```diff
@@ -796,7 +796,7 @@ module Kernel : BasicObject
   # If *const* is defined as autoload, the file name to be loaded is replaced with
   # *filename*.  If *const* is defined but not as autoload, does nothing.
   #
-  def self?.autoload: (interned _module, String filename) -> NilClass
+  def self?.autoload: (interned const, path filename) -> nil
```

Comment:
if we _ever_ decide to use `const` as a keyword we might want to change this to `constant`?